src/components/graph-view.js

```diff
@@ -194,13 +194,11 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
   componentDidMount() {
     const { initialBBox, zoomDelay, minZoom, maxZoom } = this.props;
 
-    if (this.viewWrapper.current) {
-      document.addEventListener('keydown', this.handleWrapperKeydown);
-      this.viewWrapper.current.addEventListener(
-        'click',
-        this.handleDocumentClick
-      );
-    }
+    document.addEventListener('keydown', this.handleKeyDown);
+    document.addEventListener('mousedown', this.handleDocumentClick, {
+      capture: true,
+      passive: true,
+    });
```

Comment:
Our outside-click detection has to run before child elements have a chance to stopPropagation, otherwise our pseudo-focus will get stuck on. Mousedown + capture phase runs before anything else.

Reply:
With a simple click handler, d3 calls stopPropagation on mousedown when clicking a node, so in the multiple graphs case you can click a node in each graph and end up with both graphs pseudo-focused (delete key will affect both graphs, etc.)